### PR TITLE
Implementation of Payment Get-Started UI

### PR DESCRIPTION
**Pull request by asapconet**
### Components
-  Created a reusable 'GetStartedCard' components for  for the page
### Main UI 
- Ensured full UI responsiveness across all screens **[1024 & 768]** especially.
- Created the UI for pagination 

### UI for Desktop screen
image

### UI for mobile
image
image

### Tablet view
image

**NB** this page has issues with the side/headbar and contacts have been made with resolve the resolve the issue
Attached to this is a link to the loom video explaining visually: https://www.loom.com/share/443c774092be45a8adb8e1a7a1614bfd

### CODE REVIEW

File: components/GetstartedCard/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// .GetStarted {
```
Comment (Frontend-io) [inline components/GetstartedCard/index.module.scss:1]: Please remove dead codes

File: components/GetstartedCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  completeYourIntegrationData,
+  startIntegratingData,
+  testAndGoData,
+} from './data'
+import LzLink from 'components/UI/Link'
+import LinkIcon from 'public/icons/link-icon'
+import ArrowRightIcon from 'public/icons/arrow-right-icon'
+
+// card with Icon component
+export const LinkedCard = ({ desc, title, route }) => {
```
Comment (Frontend-io) [inline components/GetstartedCard/index.tsx:13]: Please add the types to the props

File: components/GetstartedCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            <ArrowRightIcon />
+          </span>
+        </div>
+      </LzLink>
+    </div>
+  )
+}
+
+// card without Icon component
+export const PlainCard = ({ desc, title, route }) => {
```
Comment (Frontend-io) [inline components/GetstartedCard/index.tsx:36]: Please add the types to the props

File: components/GetstartedCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    >
+      {startIntegratingData.map(({ title, desc, route }) => {
+        return <LinkedCard key={route} title={title} desc={desc} route={route} />
+      })}
+    </section>
+  )
+}
+
+// 2 Complet your integration
+export function CompleteIntegration(props: any): JSX.Element {
```
Comment (Frontend-io) [inline components/GetstartedCard/index.tsx:76]: Please add the types to the props

File: components/Header/Header.module.scss [hunk 5]
```diff
@@ -1,5 +1,9 @@
...
-    @media (min-width: 1024px) {
-        box-shadow: 0px 0px 1px rgba(12, 26, 75, 0.24), 0px 3px 8px -1px rgba(50, 50, 71, 0.05);
-    }
-}
\ No newline at end of file
+  @media (min-width: 1024px) {
+    box-shadow: 0px 0px 1px rgba(12, 26, 75, 0.24),
+      0px 3px 8px -1px rgba(50, 50, 71, 0.05);
+  }
+  @media (max-width: 768px) {
```
Comment (Frontend-io) [inline components/Header/Header.module.scss:6]: Please clear redundant code

File: components/Sidebar/index.tsx [hunk 6]
```diff
@@ -76,10 +77,11 @@ const Sidebar = ({ isOpen }: IProps): JSX.Element => {
               )
             })
           }
+          <LzButton className='w-full sm:hidden' size='lg'><LzLink to='/'> Dashboard</LzLink></LzButton>
```
Comment (Frontend-io) [inline components/Sidebar/index.tsx:80]: By the way, you can actually make the button a link by passing the url via the `asLink` props.

File: components/GetstartedCard/index.tsx [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+import ArrowRightIcon from 'public/icons/arrow-right-icon'
+
+// card with Icon component
+export const LinkedCard = ({ desc, title, route }) => {
+  return (
+    <div key={route}>
+      <LzLink className='my-2' to={route}>
+        <div
+          key={route}
+          className='bg-white text-sm mb-2 p-6 md:pt-9 drop-shadow-xs rounded-8 lg:rounded-8 h-[210px] lg:w-92 z-10'
```
Comment (Frontend-io) [inline components/GetstartedCard/index.tsx:19]: Please check this file and you'll see how we have setup typography to help us easily render texts easily. https://github.com/LazerPay-Finance/Lazerpay-documentation-v2/blob/main/styles/typography.scss

We don't pass the size or font directly: see this file for how its been used https://github.com/LazerPay-Finance/Lazerpay-documentation-v2/blob/a84ba68dce671c883c2872c1d53d58815474a843/components/QuickLinks/index.tsx#L22

Please update other places you entered the font weight or size directly(eg `text-sm` or `font-bold` :))

**Review by Frontend-io — CHANGES_REQUESTED** [review]
Nice job man. Left some couple of comments, once resolved, please ping me again to take a look and we're good.

File: components/GetstartedCard/index.module.scss (hunk 1 above)
Comment (asapconet) [reply components/GetstartedCard/index.module.scss:1]: done!

File: components/GetstartedCard/index.tsx (hunk 3 above)
Comment (asapconet) [reply components/GetstartedCard/index.tsx:36]: done!

File: components/Header/Header.module.scss (hunk 5 above)
Comment (asapconet) [reply components/Header/Header.module.scss:6]: done!

File: components/GetstartedCard/index.tsx (hunk 7 above)
Comment (asapconet) [reply components/GetstartedCard/index.tsx:19]: fixed

**Comment by Frontend-io** [comment]
Looks good!

**Pull request merged**